internal/common/config/clientconfig.go

```diff
@@ -34,3 +62,88 @@ func (c DisasterConfig) GetClientDisasterConfig() ClientDisasterConfig {
 		),
 	}
 }
+
+// GetClientIIGOConfig gets ClientIIGOConfig
+func (c IIGOConfig) GetClientIIGOConfig() ClientIIGOConfig {
+	return ClientIIGOConfig{
+		// Executive branch
+		GetRuleForSpeakerActionCost: getSelectivelyVisibleResources(
```

Comment:
You dont need to call a function just use 
`GetRuleForSpeakerActionCost: c.GetRuleForSpeakerActionCost`

Same for everything below, unless you do intend to to replace `true` with another parameter

Reply:
Ohhh, but then the type isn't SelectiveVisibility like the other parameters that are passed. Is that ok?

Reply:
Ok done I get you now